modules/serverless-send-events-kn.adoc

```diff
@@ -37,7 +37,7 @@ $ kn event send \
     --field player.game=2345 \
     --field points=456 \
     --type org.example.gaming.foo \
-    --to-url http://ce-api.foo.example.com/
+    --to http://ce-api.foo.example.com/
 ----
 +
 The following example shows sending an event to an in-cluster resource:
```

Comment:
The `--to` format had changed. The 53 line isn't true anymore:

```diff
-    --to Service:serving.knative.dev/v1:event-display
+    --to ksvc:event-display
```

The format should be explained. It is the same as described here: https://docs.openshift.com/serverless/1.35/cli_tools/eventing_cli/kn-source.html#specifying-sink-flag-kn_kn-source